### PR TITLE
[Enhancement #71] Populate published status of a term when it is loaded.

### LINKED ISSUE #71: Provide info about whether term has been published
As a user, I need to be able to determine whether a term exists only in the current workspace or whether it has been published and the workspace version is just a working copy of it.

### PR DESCRIPTION
**Pull request by ledsoft**
Resolves #71 

See also https://github.com/opendata-mvcr/termit-ui/pull/96

### CODE REVIEW

**Comment by ledsoft** [comment]
[Prozkoumat způsoby, jak měnit label pojmu v TermIt](https://trello.com/c/XOUHRzps/345-prozkoumat-zp%C5%AFsoby-jak-m%C4%9Bnit-label-pojmu-v-termit)

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/TermDao.java [hunk 1]
```diff
@@ -406,20 +409,30 @@ private void loadSubTermInfo(Term parent, Set<URI> graphs) {
...
+                .setParameter("graphs", graphs)
+                .setParameter("labelLang", config.get(ConfigParam.LANGUAGE))
+                .getResultStream();
         // Use LinkedHashSet to preserve term order
         parent.setSubTerms(subTermsStream.collect(Collectors.toCollection(LinkedHashSet::new)));
     }
 
+    private boolean isPublished(Term term) {
+        // Note that this can be moved into Term once SPARQL-based attributes (https://github.com/kbss-cvut/jopa/issues/65) are supported
+        return em.createNativeQuery("ASK WHERE {" +
```
Comment (psiotwo) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/TermDao.java:426]: maybe unique inScheme link checking would be more efficient ATM.

**Review by psiotwo — DISMISSED** [review]
lgtm, see comment

**Comment by ledsoft** [comment]
Can't wait for the review forever. Merging.

**Pull request merged**